src/neuroconv/datainterfaces/ophys/scanimage/scanimageimaginginterfaces.py

```diff
@@ -730,8 +786,13 @@ def get_scanimage_major_version(scanimage_metadata: dict) -> str:
 
     Returns
     -------
-    version : str
+    str
```

Comment:
```suggestion
    vesion: str
```
I think this was fine.